packages/core/src/external/carequality/utils.ts

```diff
@@ -34,3 +34,51 @@ export function cleanXml(xml: string): string {
   xml = xml.replace(/\\"/g, '"');
   return xml;
 }
+
+interface ParsedContentType {
+  boundary: string;
+  startToken: string;
+}
+
+function parseContentTypeHeader(header: string): ParsedContentType | undefined {
```

Comment:
Why is this stuff going in core? AFAIK this it all temporary stuff for the IHE lambda?

Reply:
yah I just put all lambda code into core so I can test it local.